Makefile

```diff
@@ -87,58 +80,46 @@ output/bin/embedded-cluster-release-builder:
 	go build -o output/bin/embedded-cluster-release-builder e2e/embedded-cluster-release-builder/main.go
 
 .PHONY: embedded-release-onmerge
-embedded-release-onmerge: embedded-cluster-linux-amd64 output/tmp/release-onmerge.tar.gz output/bin/embedded-cluster-release-builder
+embedded-release-onmerge: embedded-cluster output/tmp/release-onmerge.tar.gz output/bin/embedded-cluster-release-builder
 	./output/bin/embedded-cluster-release-builder output/bin/embedded-cluster output/tmp/release-onmerge.tar.gz output/bin/embedded-cluster
 
 .PHONY: embedded-release-onpr
-embedded-release-onpr: embedded-cluster-linux-amd64 output/tmp/release-onpr.tar.gz output/bin/embedded-cluster-release-builder
+embedded-release-onpr: embedded-cluster output/tmp/release-onpr.tar.gz output/bin/embedded-cluster-release-builder
 	./output/bin/embedded-cluster-release-builder output/bin/embedded-cluster output/tmp/release-onpr.tar.gz output/bin/embedded-cluster
 
 .PHONY: static
-static: pkg/goods/bins/embedded-cluster/kubectl-preflight \
-	pkg/goods/bins/k0sctl/k0s-$(K0S_VERSION)
-
-.PHONY: static-linux-amd64
-static-linux-amd64: pkg/goods/bins/embedded-cluster/kubectl-linux-amd64 pkg/goods/bins/embedded-cluster/k0sctl-linux-amd64 pkg/goods/bins/embedded-cluster/kubectl-support_bundle-linux-amd64
-
-.PHONY: embedded-cluster-linux-amd64
-embedded-cluster-linux-amd64: static static-linux-amd64
+static: pkg/goods/bins/k0s \
+	pkg/goods/bins/kubectl-preflight \
+	pkg/goods/bins/kubectl \
+	pkg/goods/bins/kubectl-support_bundle
+	
+.PHONY: embedded-cluster
+embedded-cluster: static
```

Comment:
IMO this should still be embedded-cluster-linux-amd64 - it's still being built for linux-amd64 even if the k0sctl-linux-amd64 binary isn't included

Reply:
As you wish. Done.